### DISCUSSION TITLE
It is recommended to add a setting to control whether the date_updated field is updated

### DISCUSSION THREAD

**Discussion by cdwmhcc** [directus/directus#9030] [Feature Requests]
### Scene 1:

I created a collection `special_article`, added a field `article_id`, and related the article, the relationship is o2m.

I selected articles in the collection `special_article` and sorted them. After I saved, the `date_updated` of the selected articles were all updated, which caused me a lot of trouble. For a long time, when I needed to associate articles, I could only directly connect to the database to make changes.

### Scene 2:

I want to add read count function to my collection `article`, I created a field `total_read`, When the user views the article,`total_read` + 1, This will also update the `date_updated` field.

### My suggestion:

In fact, this problem has been bothering me for several years. I very much hope that the Directus team can consider adding a setting to control which fields do not affect the `date_updated` when updating.

**Comment by joselcvarela** [comment]
Hello @cdwmhcc 
Are you relying on `date_updated` field to sort the items? My suggestion is to create a specific field (like `sort`) for that to avoid those problems.

**Reply by cdwmhcc** [reply]
Thank you for your reply.Yes, I rely on it for sorting. Creating other fields or using sort is not a good solution for me.

**Reply by joselcvarela** [reply]
How about create a custom hook `items.update.before`, read `date_updated` from record in database, and pass it to the payload?

**Reply by cdwmhcc** [reply]
Thank you!Yes, this is how I solved it in the v8 version. But this is obviously not a good solution. I don't like to directly manipulate the database. Maybe I don't understand it deeply.

**Reply by joselcvarela** [reply]
Yes, I know it's not ideal. Just want to give a solution in the meantime.

**Comment by benhaynes** [comment]
The solution here will likely be to add a Collection setting that stores a CSV of blacklisted fields that should not be considered updates. For instance, I think we might include `sort` by default here, since changing an item's order probably shouldn't trigger a revision... THOUGH, it should still trigger the update hook, so that webhooks know to fire.

An interesting scope to solve.

**Reply by rijkvanzanten** [reply]
Also, the name `sort` is user configurable, so not necessarily something we'd want to hardcode in 🤔

**Reply by cdwmhcc** [reply]
Yeah!

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
